Pr/arctica greeter remote howto

Korn20000 opened the pull request:
As agreed the .md file has been modified/corrected so it does not contain any hmtl styles. The figures has been send separately, although they have not been set as relative URL's in the .md file. This is because I assumed that the figures should have been send first before they could have been added to the .md file.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -0,0 +1,258 @@
+Next step choose **Erase Disk** and press Install Now.
+If it asks you **“Write the changes to disk?”**, ignore it and press continue.
+
+![2](https://github.com/Korn20000/Testing/blob/master/img/2.png?raw=true)
```
Please replace absolute URLs by relative / local URLs. This comment stands for all further locations where href URLs are used.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -0,0 +1,258 @@
+
+Now that the **“Terminal”** is open, run the following commands:
+
+- **sudo -i** 
```
for command line instructions, better use three backticks (as block start marker) and another three backticks (as block end markers). The layout engine will then create a code box around the text.

See this example:

```
apt-get install markdown-editor
```

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -0,0 +1,258 @@
+
+Type following:
+
+- **nano /etc/apt/sources.list**
```
backticks here (and many more places, investigate yourself, please)

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -0,0 +1,258 @@
+
+Head to the bottom and press enter to insert new line and type following:
+
+-	**deb http://packages.arctica-project.org/ubuntu-nightly xenial main**
```
more backticks here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -0,0 +1,258 @@
+
+Go back to the folder where the 4 repositories were created using the command “**cd ..**” and install the created packages:
+
+-	**sudo dpkg -i** ***.deb**
```
too many asterisks here... replace by backtick box

Korn20000 replied:
So basically I need to make backstick on every comment :).

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -1,4 +1,4 @@
-## Arctica Greeter - Installation Guide for Ubuntu 16.04 LTS (Latest version)
+## Arctica Greeter - Installation Guide for Ubuntu 16.04 LTS (Latest version
```
typo. missing ")" at EOL

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
-
-- **`nano /etc/apt/sources.list`**
+```
+nano /etc/apt/sources.list
````
Prepent "$" or "#", depending on wether this is done as root or user.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
 
--	**`apt-get install git`**
+```
+apt-get install git
````
Prepent "$" or "#", depending on wether this is done as root or user.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
 
--	**`mkdir arctica-greeter`**
+```
+mkdir arctica-greeter
````
Prepent "$" or "#", depending on wether this is done as root or user.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
 
--	**`cd arctica-greeter`**
+```
+cd arctica-greeter
````
Prepent "$" or "#", depending on wether this is done as root or user.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
--	**`git clone https://github.com/ArcticaProject/lightdm-remote-session-x2go/`**
--	**`git clone https://github.com/ArcticaProject/remote-logon-service/`**
+```
+git clone https://github.com/ArcticaProject/arctica-greeter/
````
Prepent "$" or "#", depending on wether this is done as root or user.

No empty lines inbetween the commands, please.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+# apt-get install debhelper
+# apt-get install dh-autoreconf
+# etc. etc. etc. 
+```
````
turn into a one liner...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+# apt-get install dh-autoreconf
+# apt-get install fonts-droid-fallback
+# apt-get install gnome-common
+# apt-get install gnome-settings-daemon-dev
```
turn into a one liner...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+# apt-get install xvfb
+# apt-get install fonts-cantarell
+# apt-get install fonts-cabin
+# apt-get install gnome-settings-daemon
```
turn into a one liner...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+# apt-get install libssh-dev 
+# apt-get install cdbs 
+# apt-get install socat
+# apt-get install pyhoca-cli
```
turn into a one liner...

sunweaver replied:
make sure, you _only_ list packages from Build-Depends: field in debian/control of the libpam-x2go package. Please check all other dependencies, too.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+cd libpam-x2go
+cd lightdm-remote-session-x2go
+cd remote-logon-service
+```
````
the debuild -uc -us commands for the other folders are missing. Please add that, e.g.

```
$ cd libpam-x2go && debuild -uc -us && cd ..
```

Then you don't have to mention the "cd .." in the below text anymore.

sunweaver replied:
don't forget adding "#" or "$". Hint: package builds are done as "user".

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
 
 Open the following file, by typing:
+```
+nano /etc/lightdm/lightdm.conf.d/90-arctica-greeter.conf
````
"#" or "$"?

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
-
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"`CTRL+C`"**.
+```
+pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY
````
"$" or "#"

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
 
--	**`nano /etc/environment`**
+```
+nano /etc/environment
````
"#" or "$"

There are more occurences of this issue below, please find and fix them yourself.

Korn20000 commented:
Just wanna be sure that I managed to commit correctly last time???





-------- Original message --------
From: Mike Gabriel <notifications@github.com>
Date: 20/01/2017 14:54 (GMT+01:00)
To: ArcticaProject/ArcticaDocs <ArcticaDocs@noreply.github.com>
Cc: Kornelius Pasztor <korn_20000@hotmail.com>, Author <author@noreply.github.com>
Subject: Re: [ArcticaProject/ArcticaDocs] Pr/arctica greeter remote howto (#3)


@sunweaver requested changes on this pull request.

We are getting somewhere. Good. But I still found issues. Please fix.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

> @@ -1,4 +1,4 @@
-## Arctica Greeter - Installation Guide for Ubuntu 16.04 LTS (Latest version)
+## Arctica Greeter - Installation Guide for Ubuntu 16.04 LTS (Latest version


typo. missing ")" at EOL

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 ## Stage three

 While you are still logged in as Admin/Superuser/ROOT you will need to add the repository.

 Type following:
-
-- **`nano /etc/apt/sources.list`**
+```
+nano /etc/apt/sources.list


Prepent "$" or "#", depending on wether this is done as root or user.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
--      **`apt-get install git`**
+```
+apt-get install git


Prepent "$" or "#", depending on wether this is done as root or user.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 When the installation of Git is done, you will need to create a folder in order to clone git so they are collected in one folder. Therefore you need to type following command so you can create a folder:

--      **`mkdir arctica-greeter`**
+```
+mkdir arctica-greeter


Prepent "$" or "#", depending on wether this is done as root or user.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 Now open the folder that you just created by using following command:

--      **`cd arctica-greeter`**
+```
+cd arctica-greeter


Prepent "$" or "#", depending on wether this is done as root or user.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 Now you are able to clone the 4 repositories from ArcticaProject, so type following lines one by one:

--      **`git clone https://github.com/ArcticaProject/arctica-greeter/`**
--      **`git clone https://github.com/ArcticaProject/libpam-x2go/`**
--      **`git clone https://github.com/ArcticaProject/lightdm-remote-session-x2go/`**
--      **`git clone https://github.com/ArcticaProject/remote-logon-service/`**
+```
+git clone https://github.com/ArcticaProject/arctica-greeter/


Prepent "$" or "#", depending on wether this is done as root or user.

No empty lines inbetween the commands, please.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
-Install libpam-x2go dependencies, use the same command **"`apt-get install`"** followed by the dependencies:
+```
+# apt-get install debhelper
+# apt-get install dh-autoreconf
+# etc. etc. etc.
+```


turn into a one liner...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
--      **`libgtest-dev`**
--      **`libpam0g-dev`**
--      **`libssh-dev`**
--      **cdbs**
--      **`socat`**
--      **`pyhoca-cli`**
+The following packages that need to be installed:
+```
+# apt-get install debhelper
+# apt-get install dh-autoreconf
+# apt-get install fonts-droid-fallback
+# apt-get install gnome-common
+# apt-get install gnome-settings-daemon-dev


turn into a one liner...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

> +```
+
+Now carry on with `apt-get install` followed by the command.
+
+```
+# apt-get install libayatana-ido3-0.4-dev
+# apt-get install libayatana-indicator3-dev
+# apt-get install libcanberra-dev
+# apt-get install libgnome-desktop-3-dev
+# apt-get install libgtk-3-dev
+# apt-get install liblightdm-gobject-1-dev
+# apt-get install valac
+# apt-get install xvfb
+# apt-get install fonts-cantarell
+# apt-get install fonts-cabin
+# apt-get install gnome-settings-daemon


turn into a one liner...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

> +# apt-get install valac
+# apt-get install xvfb
+# apt-get install fonts-cantarell
+# apt-get install fonts-cabin
+# apt-get install gnome-settings-daemon
+```
+
+Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
+
+```
+# apt-get install libgtest-dev
+# apt-get install libpam0g-dev
+# apt-get install libssh-dev
+# apt-get install cdbs
+# apt-get install socat
+# apt-get install pyhoca-cli


turn into a one liner...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

> +# apt-get install valac
+# apt-get install xvfb
+# apt-get install fonts-cantarell
+# apt-get install fonts-cabin
+# apt-get install gnome-settings-daemon
+```
+
+Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
+
+```
+# apt-get install libgtest-dev
+# apt-get install libpam0g-dev
+# apt-get install libssh-dev
+# apt-get install cdbs
+# apt-get install socat
+# apt-get install pyhoca-cli


make sure, you only list packages from Build-Depends: field in debian/control of the libpam-x2go package. Please check all other dependencies, too.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
--      **`cd libpam-x2go`**
--      **`cd lightdm-remote-session-x2go`**
--      **`cd remote-logon-service`**
+```
+cd libpam-x2go
+cd lightdm-remote-session-x2go
+cd remote-logon-service
+```


the debuild -uc -us commands for the other folders are missing. Please add that, e.g.

$ cd libpam-x2go && debuild -uc -us && cd ..


Then you don't have to mention the "cd .." in the below text anymore.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
--      **`cd libpam-x2go`**
--      **`cd lightdm-remote-session-x2go`**
--      **`cd remote-logon-service`**
+```
+cd libpam-x2go
+cd lightdm-remote-session-x2go
+cd remote-logon-service
+```


don't forget adding "#" or "$". Hint: package builds are done as "user".

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 ## Stage six

 Open the following file, by typing:
+```
+nano /etc/lightdm/lightdm.conf.d/90-arctica-greeter.conf


"#" or "$"?

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 Now test the Connection to the server and type following:

-- **`pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY`**
-
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"`CTRL+C`"**.
+```
+pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY


"$" or "#"

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>:

>
 Now edit the environment file, open the file by typing:

--      **`nano /etc/environment`**
+```
+nano /etc/environment


"#" or "$"

There are more occurences of this issue below, please find and fix them yourself.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-17677954>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AVTsu0T4s3YV-BZeFTMt_Lr6-DVme-NRks5rULyagaJpZM4Lm7mN>.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-```
-# apt-get upgrade
-```
+- **`# apt-get upgrade`**
````
the formating is bogus again, should be:

```
# apt-get upgrade
```

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-nano /etc/apt/sources.list
-```
+
+- **`# nano /etc/apt/sources.list`**
````
same here

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-```
-deb http://packages.arctica-project.org/ubuntu-nightly xenial main
-```
+- **`# deb http://packages.arctica-project.org/ubuntu-nightly xenial main`**
````
same here, of course this is not a shell command, so "#" does not make sense here!!! Remote "'#" again.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-```
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8680F5598DE3101
-```
+- **`#apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8680F5598DE3101`**
````
same bogus format here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-apt-get install git
-```
+
+- **`# apt-get install git`**
````
and here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-mkdir arctica-greeter
-```
+
+- **`# mkdir arctica-greeter`**
````
and here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-```
-cd arctica-greeter
-```
+- **`# cd arctica-greeter`**
````
and here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -42,82 +42,67 @@ $ sudo -i
-git clone https://github.com/ArcticaProject/remote-logon-service/
-```
+- **`# git clone https://github.com/ArcticaProject/remote-logon-service/`**
+ 
````
and here ... (sigh)

sunweaver replied:
and... git clone has to be run by the user, not by root (so use "$")

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-# apt-get install dh-autoreconf
-# etc. etc. etc. 
-```
+- **`# apt-get install debhelper dh-autoreconf etc. etc. etc.`** 
````
and here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-# apt-get install gnome-settings-daemon-dev
-```
+
+- **`# apt-get install debhelper dh-autoreconf fonts-droid-fallback gnome-common gnome-settings-daemon-dev`**
````
and here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-# apt-get update
-```
+
+- **`# apt-get update`**
````
and here...

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-# apt-get install fonts-cabin
-# apt-get install gnome-settings-daemon
-```
+- **`# apt-get install libayatana-ido3-0.4-dev libayatana-indicator3-dev libcanberra-dev libgnome-desktop-3-dev libgtk-3-dev liblightdm-gobject-1-dev valac xvfb fonts-cantarell fonts-cabin
````
for better readability, use this way of putting it here:

```
# apt-get install libayatana-ido3-0.4-dev libayatana-indicator3-dev \
                  libcanberra-dev libgnome-desktop-3-dev libgtk-3-dev \
                  liblightdm-gobject-1-dev valac xvfb fonts-cantarell \
                  fonts-cabin gnome-settings-daemon
```

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-# apt-get install socat
-# apt-get install pyhoca-cli
-```
+- **`# apt-get install libgtest-dev libpam0g-dev libssh-dev cdbs socat pyhoca-cli`**
````
use the backslash "\" format here, too (as shown in prev comment)

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-```
-# apt-get install dh-apparmor
-```
+- **`#apt-get install dh-apparmor`**
````
again, bogus format, use 3 backticks, line break, command, line break, 3 backticks

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-# apt-get install libnm-glib-dev
-# apt-get install libsoup2.4-dev
-```
+- **`# apt-get install dbus-test-runner libdbustest1-dev libgcrypt11-dev libjson-glib-dev libnm-glib-dev
````
bogus format, use backslashes and line-wrap the package names (see example above)

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-```
-# apt-get install devscripts
-```
+- **`# apt-get install devscripts`**
````
bogus format

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-```
-cd arctica-greeter
-```
+- **`# cd arctica-greeter`**
````
same here

sunweaver replied:
run as user

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-```
-debuild –uc –us
-```
+- **`# debuild –uc –us`**
````
same...

sunweaver replied:
debuild needs to be run as "user", so "$"

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-cd lightdm-remote-session-x2go
-cd remote-logon-service
-```
+- **`# cd libpam-x2go && debuild -uc -us && cd lightdm-remote-session-x2go && debuild -uc -us && cd remote-logon-service && debuild -uc -us`**
````
and here...

sunweaver replied:
run as user, so "$"

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-```
-sudo dpkg -i *.deb
-```
+- **`# sudo dpkg -i *.deb`**
````
and here...

sunweaver replied:
Run use user (with sudo), so "$"

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
```diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
 **"CTRL+O"**  and press enter to save the file, afterwards exit it by pressing **"CTRL+X"**.
 
-Type **"exit"** in order to logout as Admin, type **"exit"** again so it will close the Terminal. Restart the system and you will see Arctica Greeter as your login screen.
+Type **`exit`** in order to logout as Admin, type **`exit`** again so it will close the Terminal. Restart the system and you will see Arctica Greeter as your login screen.
```
**` does not have a meaning in markdown afaik, so don't use it or send me a reference that explains its usage...

Korn20000 replied:
WTH, I must have done something wrong with git again >:( .........
I have the absolute corrected version on my notebook right here in front of me, I even used the webpage u provided me!

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
-```
-# apt-get install thin-client-config-agent
-```
+- **`# apt-get install thin-client-config-agent`**
````
same here... (there are 11 or so other occurrences, please re-fix them all) (they used to be ok in the previous comment accepted the trailing "#" or "$" (which you now added, but with that you went back to the broken code quoting format of yours).

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
-pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY
-```
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"CTRL+C"**.
+- **`# pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY`**
````
Perform the pyhoca-cli test as user, please, not as root. So "$".

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
-```
-sudo su
-```
+- **`$ sudo -i`**
````
Don't run the tests as root here. The thin-client-config-agent works under normal user privileges.

Furthermore, I switched over to our own script here. The thin-client-config-agent is +/- unmaintained by Canonical there days, so I forked it, too. See: 
https://github.com/ArcticaProject/remote-logon-config-agent

Please put remote-logon-config-agent here, instead of thin-client-config-agent.

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
-```
-echo <password> | thin-client-config-agent <username>
-```
+- **`# echo <password> | thin-client-config-agent <username>`**
````
remote-logon-config-agent also here

sunweaver commented on ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md:
````diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
-```
-echo testuser | thin-client-config-agent testuser
-```
+- **`# echo testuser | thin-client-config-agent testuser`**
````
also here

Korn20000 commented:
Hi Mike I have absolutely no clue what is going on here, like srsly I have the corrected version right here in front of me, why it's doing that I have no clue. If needed I can give u a share screen of it.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------
From: Mike Gabriel <notifications@github.com>
Date: 24/01/2017 10:26 (GMT+01:00)
To: ArcticaProject/ArcticaDocs <ArcticaDocs@noreply.github.com>
Cc: Kornelius Pasztor <korn_20000@hotmail.com>, Author <author@noreply.github.com>
Subject: Re: [ArcticaProject/ArcticaDocs] Pr/arctica greeter remote howto (#3)


@sunweaver requested changes on this pull request.

Next iteration. With the last commit, you broke the 3-backticks format of all code/command/config file quoting blocks. Please re-do the previous format of those passages. And keep "#" and "$", if not commented otherwise.

Except from the above, I added more comments inline, please see
https://github.com/ArcticaProject/ArcticaDocs/pull/3/files/c6a501449b08743a532bb8b3fec34b45d2250d56..020eb438f0ba629d737f4e90b79e961e7962f556

for details...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 This will give you the latest available package versions in the repositiory.

 When it is done with getting the latest package version, you will need to download and install them. Then...

-```
-# apt-get upgrade
-```
+- **`# apt-get upgrade`**


the formating is bogus again, should be:

# apt-get upgrade


________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>  ## Stage three

 While you are still logged in as Admin/Superuser/ROOT you will need to add the repository.

 Type following:
-```
-nano /etc/apt/sources.list
-```
+
+- **`# nano /etc/apt/sources.list`**


same here

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Head to the bottom and press enter to insert new line and type following:

-```
-deb http://packages.arctica-project.org/ubuntu-nightly xenial main
-```
+- **`# deb http://packages.arctica-project.org/ubuntu-nightly xenial main`**


same here, of course this is not a shell command, so "#" does not make sense here!!! Remote "'#" again.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now write out the repository by pressing **"CTRL+O"**, it will ask you to give the **File Name to Write** press enter to that.

 After you have saved it, press **"CTRL+X"** to exit the repository.

 You should now be back in the Terminal, type the following command to fetch the key from a server:

-```
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8680F5598DE3101
-```
+- **`#apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8680F5598DE3101`**


same bogus format here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 It will tell you that a **RSA: 1 is imported** if it succeeded to import the key.

 Next you need to install git, type following command, it will ask you if you want to continue type **"y"** and press enter:

-```
-apt-get install git
-```
+
+- **`# apt-get install git`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 When the installation of Git is done, you will need to create a folder in order to clone git so they are collected in one folder. Therefore you need to type following command so you can create a folder:

-```
-mkdir arctica-greeter
-```
+
+- **`# mkdir arctica-greeter`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now open the folder that you just created by using following command:

-```
-cd arctica-greeter
-```
+- **`# cd arctica-greeter`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
-git clone https://github.com/ArcticaProject/remote-logon-service/
-```
+- **`# git clone https://github.com/ArcticaProject/remote-logon-service/`**
+


and here ... (sigh)

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

> @@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For

 **Example:**

-```
-# apt-get install debhelper
-# apt-get install dh-autoreconf
-# etc. etc. etc.
-```
+- **`# apt-get install debhelper dh-autoreconf etc. etc. etc.`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 The following packages that need to be installed:
-```
-# apt-get install debhelper
-# apt-get install dh-autoreconf
-# apt-get install fonts-droid-fallback
-# apt-get install gnome-common
-# apt-get install gnome-settings-daemon-dev
-```
+
+- **`# apt-get install debhelper dh-autoreconf fonts-droid-fallback gnome-common gnome-settings-daemon-dev`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 In order to install libayatana you will need to update the packages type following:
-```
-# apt-get update
-```
+
+- **`# apt-get update`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now carry on with `apt-get install` followed by the command.

-```
-# apt-get install libayatana-ido3-0.4-dev
-# apt-get install libayatana-indicator3-dev
-# apt-get install libcanberra-dev
-# apt-get install libgnome-desktop-3-dev
-# apt-get install libgtk-3-dev
-# apt-get install liblightdm-gobject-1-dev
-# apt-get install valac
-# apt-get install xvfb
-# apt-get install fonts-cantarell
-# apt-get install fonts-cabin
-# apt-get install gnome-settings-daemon
-```
+- **`# apt-get install libayatana-ido3-0.4-dev libayatana-indicator3-dev libcanberra-dev libgnome-desktop-3-dev libgtk-3-dev liblightdm-gobject-1-dev valac xvfb fonts-cantarell fonts-cabin


for better readability, use this way of putting it here:

# apt-get install libayatana-ido3-0.4-dev libayatana-indicator3-dev \
                  libcanberra-dev libgnome-desktop-3-dev libgtk-3-dev \
                  liblightdm-gobject-1-dev valac xvfb fonts-cantarell \
                  fonts-cabin gnome-settings-daemon


________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:

-```
-# apt-get install libgtest-dev
-# apt-get install libpam0g-dev
-# apt-get install libssh-dev
-# apt-get install cdbs
-# apt-get install socat
-# apt-get install pyhoca-cli
-```
+- **`# apt-get install libgtest-dev libpam0g-dev libssh-dev cdbs socat pyhoca-cli`**


use the backslash "" format here, too (as shown in prev comment)

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Install the lightdm-remote-session-x2go dependencies, use the same command `apt-get install` followed by the dependencies:

-```
-# apt-get install dh-apparmor
-```
+- **`#apt-get install dh-apparmor`**


again, bogus format, use 3 backticks, line break, command, line break, 3 backticks

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 install remote-logon-service dependencies, use the same command `apt-get install` followed by the dependencies:

-```
-# apt-get install dbus-test-runner
-# apt-get install libdbustest1-dev
-# apt-get install libgcrypt11-dev
-# apt-get install libjson-glib-dev
-# apt-get install libnm-glib-dev
-# apt-get install libsoup2.4-dev
-```
+- **`# apt-get install dbus-test-runner libdbustest1-dev libgcrypt11-dev libjson-glib-dev libnm-glib-dev


bogus format, use backslashes and line-wrap the package names (see example above)

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 ## Stage five

 Install the devscripts, using same command `apt-get install` followed by the dependence:

-```
-# apt-get install devscripts
-```
+- **`# apt-get install devscripts`**


bogus format

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now navigate to each repository folder that was created, in order to open the folders you will need to type `cd` in front of each folder, though to go back one folder you will need to type `cd ..`

 Type following:

-```
-cd arctica-greeter
-```
+- **`# cd arctica-greeter`**


same here

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now you are in the folder here you will need to run the command:

-```
-debuild –uc –us
-```
+- **`# debuild –uc –us`**


same...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
-```
-cd libpam-x2go
-cd lightdm-remote-session-x2go
-cd remote-logon-service
-```
+- **`# cd libpam-x2go && debuild -uc -us && cd lightdm-remote-session-x2go && debuild -uc -us && cd remote-logon-service && debuild -uc -us`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Go back to the folder where the 4 repositories were created using the command `cd ..` and install the created packages:

-```
-sudo dpkg -i *.deb
-```
+- **`# sudo dpkg -i *.deb`**


and here...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>  **"CTRL+O"**  and press enter to save the file, afterwards exit it by pressing **"CTRL+X"**.

-Type **"exit"** in order to logout as Admin, type **"exit"** again so it will close the Terminal. Restart the system and you will see Arctica Greeter as your login screen.
+Type **`exit`** in order to logout as Admin, type **`exit`** again so it will close the Terminal. Restart the system and you will see Arctica Greeter as your login screen.


**` does not have a meaning in markdown afaik, so don't use it or send me a reference that explains its usage...

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 In same folder install thin-client-config-agent:

-```
-# apt-get install thin-client-config-agent
-```
+- **`# apt-get install thin-client-config-agent`**


same here... (there are 11 or so other occurrences, please re-fix them all) (they used to be ok in the previous comment accepted the trailing "#" or "$" (which you now added, but with that you went back to the broken code quoting format of yours).

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now you are in the folder here you will need to run the command:

-```
-debuild –uc –us
-```
+- **`# debuild –uc –us`**


debuild needs to be run as "user", so "$"

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Now navigate to each repository folder that was created, in order to open the folders you will need to type `cd` in front of each folder, though to go back one folder you will need to type `cd ..`

 Type following:

-```
-cd arctica-greeter
-```
+- **`# cd arctica-greeter`**


run as user

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
-git clone https://github.com/ArcticaProject/remote-logon-service/
-```
+- **`# git clone https://github.com/ArcticaProject/remote-logon-service/`**
+


and... git clone has to be run by the user, not by root (so use "$")

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
-```
-cd libpam-x2go
-cd lightdm-remote-session-x2go
-cd remote-logon-service
-```
+- **`# cd libpam-x2go && debuild -uc -us && cd lightdm-remote-session-x2go && debuild -uc -us && cd remote-logon-service && debuild -uc -us`**


run as user, so "$"

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Go back to the folder where the 4 repositories were created using the command `cd ..` and install the created packages:

-```
-sudo dpkg -i *.deb
-```
+- **`# sudo dpkg -i *.deb`**


Run use user (with sudo), so "$"

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

> @@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by

 Now test the Connection to the server and type following:

-```
-pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY
-```
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"CTRL+C"**.
+- **`# pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY`**


Perform the pyhoca-cli test as user, please, not as root. So "$".

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 Type exit and press enter this will put you back to the very beginning, log back in as an Admin/Superuser/ROOT by typing following:

-```
-sudo su
-```
+- **`$ sudo -i`**


Don't run the tests as root here. The thin-client-config-agent works under normal user privileges.

Furthermore, I switched over to our own script here. The thin-client-config-agent is +/- unmaintained by Canonical there days, so I forked it, too. See:
https://github.com/ArcticaProject/remote-logon-config-agent

Please put remote-logon-config-agent here, instead of thin-client-config-agent.

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>  type your password as you did in the beginning.

 Now type following to get information about remote desktop servers:

-```
-echo <password> | thin-client-config-agent <username>
-```
+- **`# echo <password> | thin-client-config-agent <username>`**


remote-logon-config-agent also here

________________________________

In ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>:

>
 In your case it could look like:

-```
-echo testuser | thin-client-config-agent testuser
-```
+- **`# echo testuser | thin-client-config-agent testuser`**


also here

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ArcticaProject/ArcticaDocs/pull/3#pullrequestreview-18111032>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AVTsu-zS7yo8W3N6B-wOSNRWItNKP0e_ks5rVcPRgaJpZM4Lm7mN>.

sunweaver commented:
Finally done and superceded by PR #5.
